internal/pkg/docker/dockercompose/svc.go

```diff
@@ -0,0 +1,104 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package dockercompose
+
+import (
+	"fmt"
+	"github.com/aws/copilot-cli/internal/pkg/manifest"
+	"github.com/compose-spec/compose-go/types"
+	"time"
+)
+
+func convertBackendService(service *types.ServiceConfig, port uint16) (*manifest.BackendServiceConfig, IgnoredKeys, error) {
```

Comment:
Just curious how the returned `IgnoredKeys` will be handled.

Reply:
I intend for it to be passed through up to the point where the user is interacting with the software, so that an appropriate warning message can be displayed to them, without disrupting conversion. The fatal keys, however, generate an error instead.

Reply:
👍🏼 
Just a note: let's build with conversion extensability in mind; as Docker Compose or we add/change/(deprecate) fields, it should be easy enough to adapt in the conversion code.